sdk/storage/Azure.Storage.Queues/README.md

```diff
@@ -125,6 +125,63 @@ QueueClient queue = new QueueClient(accountUri, new DefaultAzureCredential());
 
 Learn more about enabling Azure Active Directory for authentication with Azure Storage in [our documentation][storage_ad] and [our samples](#next-steps).
 
+### Message encoding
+
+This version of library does not encode message by default. V11 and prior versions base64-encoded messages by default.
+This section shows how message encoding can be customized as well as additional error handling can be provided for poison messages.
+
+#### Configuration
+
+```C# Snippet:Azure_Storage_Queues_Samples_Sample03_MessageEncoding_ConfigureMessageEncodingAsync
+QueueClientOptions queueClientOptions = new QueueClientOptions()
+{
+    MessageEncoding = QueueMessageEncoding.Base64
+};
+
+QueueClient queueClient = new QueueClient(connectionString, queueName, queueClientOptions);
+```
+
+#### Poison messages
```

Comment:
This part should probably move from the main README to Samples as it's not something we want to present every customer checking out the package.  (And ideally most customers won't have to deal with this if they're configured the same end to end.)

Reply:
Since we don't have "samples" directory with md files like Core has. I ended up moving this example to event docstring.